salt/modules/yumpkg.py

```diff
@@ -2581,6 +2581,10 @@ def group_info(name, expand=False, ignore_groups=None):
         to ``mandatory``, ``optional``, and ``default`` for accuracy, as
         environment groups include other groups, and not packages. Finally,
         this function now properly identifies conditional packages.
+    .. versionchanged:: 3006.2
```

Comment:
This seems like its actually a feature, which resolves a bug so this would need to be `3007.0`, same for the 3006.2 references below

Reply:
Sorry, had my other PR in mind, I meant `pkg.group_installed`, not `pkgrepo.absent`.

Here's the relevant block of code: https://github.com/saltstack/salt/blob/c33b562/salt/states/pkg.py#L3402-L3406